LayoutTests/fast/scrolling/mac/wheel-event-in-overflow-with-shadow-dom.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ AsyncOverflowScrollingEnabled=true ] -->
+<html>
+<head>
+    <style>
+        body {
+            height: 5000px;
+        }
+        
+        .filler {
+            width: 20px;
+            height: 250px;
+            background-color: silver;
+        }
+
+        #container {
+            height: 400px;
+            width: 400px;
+            padding: 10px;
+            border: 1px solid black;
+        }
+        
+        #target {
+            border: 1px solid gray;
+            padding: 10px;
+            height: 150px;
+            background-color: orange;
+        }
+    </style>
+    <script src="../../../resources/js-test-pre.js"></script>
```

Comment:
We usually use `<script src="../../../resources/js-test.js"></script>` in new tests rather than `pre` and `post`.